### PR TITLE
Fabian blog post & teams page update

### PR DESCRIPTION
**Pull request by siggisim**
Adds blog post for Fabian (targeting tomorrow to submit / publish) and adds Fabian to the teams page. 

Also creates an Emeriti (latin plural of Emeritus) section for BuildBuddies that have moved on.

Ping me on Slack if I should make any edits!

### CODE REVIEW

**Review by tylerwilliams — APPROVED** [review]

**Review by fmeum — APPROVED** [review]

File: website/src/pages/team.tsx [hunk 1]
```diff
@@ -120,6 +112,36 @@ const teamMembers = [
...
+    github: "pariparajuli",
+    linkedin: "pari-parajuli",
+    blurb: "Software Engineering Intern studying Computer Science at University of California, Berkeley.",
+    image: "pari.png",
+  },
+  {
+    name: "Brentley Jones",
+    email: "brentley@buildbuddy.io",
+    github: "brentleyjones",
+    twitter: "brentleyjones",
```
Comment (brentleyjones) [inline website/src/pages/team.tsx:140]: Can I request a mastodon link? 😅

Comment (siggisim) [reply website/src/pages/team.tsx:140]: Haha! Good request - will get around to this when I get some time

File: website/src/pages/team.tsx [hunk 2]
```diff
@@ -120,6 +112,36 @@ const teamMembers = [
...
+    name: "Pari Parajuli",
+    email: "pari@buildbuddy.io",
+    github: "pariparajuli",
+    linkedin: "pari-parajuli",
+    blurb: "Software Engineering Intern studying Computer Science at University of California, Berkeley.",
+    image: "pari.png",
+  },
+  {
+    name: "Brentley Jones",
+    email: "brentley@buildbuddy.io",
```
Comment (brentleyjones) [inline website/src/pages/team.tsx:138]: Do you want to remove these email addresses now?

Comment (siggisim) [reply website/src/pages/team.tsx:138]: Good call, sent https://github.com/buildbuddy-io/buildbuddy/pull/6354

**Pull request merged**